README.md

```diff
@@ -11,12 +11,12 @@ The IsThicc-Bot is a small bot to help IsThicc staff with everyday tasks. Most b
 
 ## Contributing 
 
-Feel free to fork this repo and make any edits you think would be good. An IsThicc staff member will review your pull request and either approve or deny.
+Feel free to fork this repo and make any edits you think would be beneficial. An IsThicc staff member will review your pull request and approve or deny it.
 
 ### Please note:
 
-We reserve the right to reject any pull request or edits without reason or warning. When editing any IsThicc source code you waive all rights over the code you contribute. Any attempt to claim the code as your own will be met with consequences to our full extent.
+We reserve the right to reject any pull request or edits without reason or warning. When editing any IsThicc source code, you waive all rights over the code you contribute. Any attempt to claim the code as your own will be met with consequences to the fullest extent of the law.
```

Comment:
We generally do not allow the editing of our legal information. Everything else looks good in this commit but this gives a different context to this paragraph.